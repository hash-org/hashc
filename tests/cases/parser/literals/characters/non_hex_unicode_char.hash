// run=fail, stage=parse

k := '\u{g00000}';
