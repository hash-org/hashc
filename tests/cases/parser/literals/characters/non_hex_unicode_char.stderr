error: invalid character in unicode escape sequence `g`
 --> $DIR/non_hex_unicode_char.hash:3:10
2 |   
3 |   k := '\u{g00000}';
  |            ^ here
  = info: unicode literals may only contain hexadecimal digits
